dash_auth/basic_auth.py

```diff
@@ -16,7 +16,7 @@ def is_authorized(self):
             return False
         username_password = base64.b64decode(header.split('Basic ')[1])
         username_password_utf8 = username_password.decode('utf-8')
-        username, password = username_password_utf8.split(':')
+        username, password = username_password_utf8.split(':', maxsplit=1)
```

Comment:
```suggestion
        username, password = username_password_utf8.split(':', 1)
```
For Python2 compatibility (I know I know, but we can't drop it just yet)

Reply:
Alright, it wouldn't let me commit this via GUI but it would the normal way from my computer. Thanks again @wwerst, merging 🎉